packages/neos-ui/src/apiExposureMap.js

```diff
@@ -50,26 +50,26 @@ import ModelWriter from '@ckeditor/ckeditor5-engine/src/model/writer';
 
 import ViewAttributeElement from '@ckeditor/ckeditor5-engine/src/view/attributeelement';
 import ViewContainerElement from '@ckeditor/ckeditor5-engine/src/view/containerelement';
-import ViewDocument from '@ckeditor/ckeditor5-engine/src/view/document.js';
-import ViewDocumentFragment from '@ckeditor/ckeditor5-engine/src/view/documentfragment.js';
-import ViewDocumentSelection from '@ckeditor/ckeditor5-engine/src/view/documentselection.js';
-import ViewDOMConverter from '@ckeditor/ckeditor5-engine/src/view/domconverter.js';
-import ViewEditableElement from '@ckeditor/ckeditor5-engine/src/view/editableelement.js';
-import ViewElement from '@ckeditor/ckeditor5-engine/src/view/element.js';
-import ViewEmptyElement from '@ckeditor/ckeditor5-engine/src/view/emptyelement.js';
-import ViewFiller from '@ckeditor/ckeditor5-engine/src/view/filler.js';
-import ViewMatcher from '@ckeditor/ckeditor5-engine/src/view/matcher.js';
-import ViewNode from '@ckeditor/ckeditor5-engine/src/view/node.js';
-import ViewPlaceholder from '@ckeditor/ckeditor5-engine/src/view/placeholder.js';
-import ViewPosition from '@ckeditor/ckeditor5-engine/src/view/position.js';
-import ViewRange from '@ckeditor/ckeditor5-engine/src/view/range.js';
+import ViewDocument from '@ckeditor/ckeditor5-engine/src/view/document';
+import ViewDocumentFragment from '@ckeditor/ckeditor5-engine/src/view/documentfragment';
+import ViewDocumentSelection from '@ckeditor/ckeditor5-engine/src/view/documentselection';
+import ViewDOMConverter from '@ckeditor/ckeditor5-engine/src/view/domconverter';
+import ViewEditableElement from '@ckeditor/ckeditor5-engine/src/view/editableelement';
+import ViewElement from '@ckeditor/ckeditor5-engine/src/view/element';
+import ViewEmptyElement from '@ckeditor/ckeditor5-engine/src/view/emptyelement';
+import * as ViewFiller from '@ckeditor/ckeditor5-engine/src/view/filler';
+import ViewMatcher from '@ckeditor/ckeditor5-engine/src/view/matcher';
+import ViewNode from '@ckeditor/ckeditor5-engine/src/view/node';
+import * as ViewPlaceholder from '@ckeditor/ckeditor5-engine/src/view/placeholder';
```

Comment:
Has this been tested - console.logeed ? ^^

Reply:
"looks" fine on first sight

![image](https://user-images.githubusercontent.com/85400359/194720642-24733e41-cd3a-46dd-bc49-2cdfac71dd37.png)